site/docs/4.1/components/carousel.md

```diff
@@ -281,6 +283,12 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td>true</td>
       <td>Whether the carousel should cycle continuously or have hard stops.</td>
     </tr>
+    <tr>
+      <td>touch</td>
+      <td>boolean</td>
+      <td>true</td>
```

Comment:
One thing. This is true, but maybe we should mention that it will be disabled gracefully if HammerJs isn't present?

Reply:
You're right we should add that